bundesliga_site/views.py

```diff
@@ -48,3 +36,61 @@ def get_auth_token(user):
         print ('UserSocialAuth does not exists!')
     return token
 
+
+@method_decorator(login_required, name='dispatch')
+class SelectEvents(TemplateView, LoginRequiredMixin):
+
+    """ This is the index view. Here we display all the banners that the user
+    has created """
+
+    template_name = 'organizer/select_events.html'
+
+    def _get_event(self):
+        eventbrite = Eventbrite(
+            get_auth_token(self.request.user),
+        )
+        return [
+            event
+            # Status : live, draft, canceled, started, ended, all
+            for event in eventbrite.get(
+                '/users/me/owned_events/?status=live,draft'
+            )['events']
+        ]
+
+    def post(self, *args, **kwargs):
+        # import ipdb; ipdb.set_trace()
+        events = self._get_event()
+        request_body_elements = str(self.request.body).split('event_')
+        request_body_elements.pop(0)
+        events_id = []
+        # Put in array all the events id selected
+        for event_id in request_body_elements:
+            event = event_id.split('=')
+            events_id.append(event[0])
+
+        # For each event in the api, verify if the selected ones are corrects
+        for event_in_api in events:
+            # If the event id is an event of organizer
+            if event_in_api['id'] in events_id:
+                # Verify if this event already exist
+                if not Event.objects.filter(event_id=event_in_api['id']).exists():
```

Comment:
Just a thought: The event's name might have been updated. Consider doing an 'upsert' operation here. 

https://docs.djangoproject.com/en/2.1/ref/models/querysets/#update-or-create

Reply:
It's true, we are going to research about that!